addons/mail/models/res_partner.py

```diff
@@ -64,6 +64,23 @@ def _compute_im_status(self):
         if odoobot in self:
             odoobot.im_status = 'bot'
 
+    @api.depends_context('company')
```

Comment:
You cannot depend on context for a stored field, as it implies computing it every single time it's fetched